src/types.ts

```diff
@@ -69,8 +69,19 @@ export type TrackedEvent = {
 
 export interface RequestFeedbackOptions
   extends Omit<OpenFeedbackFormOptions, "key" | "onSubmit"> {
+  /**
```

Comment:
should we annotate all types in this file? And can we reference them in the `main` module to describe those `options` and etc?